src/platform_linux/laio.c

```diff
@@ -97,7 +97,11 @@ io_handle_init(laio_handle         *io,
    } else {
       io->fd = open(cfg->filename, cfg->flags);
    }
-   platform_assert(io->fd != -1);
+   if (io->fd == -1) {
+      platform_error_log(
+         "open %s failed: %s\n", cfg->filename, strerror(errno));
```

Comment:
Minor nit suggestion. Can you rephrase the message as: ```open() '%s' failed ...```

The `()` will make it clear that a system call to some fn named `open` failed. The enclosing `''` is conventional, in my experience, to enclose string parameters so if (ever) the parameter was an empty string `""`, you'd see it immediately. (Minor stylistic suggestion.)

Reply:
Fixed on latest, thanks.